vendor/theme/templates/layouts/post-donate-page-with-recurring-ask.liquid

```diff
@@ -143,7 +154,7 @@
 	.center-content__one-column{
 		max-width: 100% !important;
 	}
-	.fr-copy, .de-copy, .es-copy, .pt-copy{
+	.fr-copy, .de-copy, .es-copy, .pt-copy, .nl-copy, .ar-copy {
```

Comment:
I discovered this while testing; the NL copy is showing for the English language (so the members are seeing both English and Dutch).